docs/docs/migrations.md

```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
     allowNull: false
   }
 )
+
+// or with an explicit schema:
+
+queryInterface.addColumn({
+    tableName: 'nameOfAnExistingTable',
+    schema: 'nameOfAnExistingSchema'
+  },
+  'nameOfTheNewAttribute',
+  Sequelize.STRING
+)
+
 ```
 
-### removeColumn(tableName, attributeName, options)
+### removeColumn(tableNameOrOptions, attributeName, options)
 
 This method allows deletion of a specific column of an existing table.
 
 ```js
-queryInterface.removeColumn('Person', 'signature')
+queryInterface.removeColumn('nameOfAnExistingTable', 'nameOfAnExistingAttribute')
```

Comment:
Don't change it to `nameOfAnExistingTable` and `nameOfAnExistingAttribute`, Its already clear from method signature. Same goes for schema example below